engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetBlend.java

```diff
@@ -15,48 +15,55 @@
  */
 package org.terasology.rendering.dag.stateChanges;
 
+import static org.lwjgl.opengl.GL11.GL_BLEND;
 import org.terasology.rendering.dag.RenderPipelineTask;
 import org.terasology.rendering.dag.StateChange;
+import org.terasology.rendering.dag.tasks.DisableCapabilityTask;
 import org.terasology.rendering.dag.tasks.EnableCapabilityTask;
 
 /**
  * TODO: Add javadocs
  */
-public class EnableCapability implements StateChange {
-    private int capability;
-    private EnableCapabilityTask task;
+public final class SetBlend extends SetCapability {
```

Comment:
These are the capabilities I am referring to:

http://feedback.wildfiregames.com/report/opengl/

A prettier but more Apple-centric table is available here:

https://developer.apple.com/opengl/capabilities/

Finally, for use of the term "capabilities" in LWJGL and Terasology see:

`org.terasology.engine.subsystems.lwjgl.LwjglGraphics.checkOpenGL()`

In line with comments made later I recommend EnableBlend() instead of EnableBlend(true) and then using EnableBlend(false) as the default instance, which should be used only internally rather than in nodes. Alternatively you could use DisableBlend() for the default instance but I'm not sure if this would make things unnecessarily complicated when the checks are made to verify redundancy. 

I don't see how a SetBlend class used as a base would be beneficial. But I do see the benefit of using the term Enable/Disable for boolean parameters and Set for non-boolean parameters.